web-spi/src/main/java/ws/ament/hammock/web/spi/ServletDescriptor.java

```diff
@@ -30,7 +30,7 @@
     private final String[] value;
     private final String[] urlPatterns;
     private final int loadOnStartup;
-    private final WebInitParam[] initParams;
+    private final transient WebInitParam[] initParams;
```

Comment:
Annotations should be serializable without an issue.  See also http://stackoverflow.com/questions/15904773/are-java-annotations-serializable